### PR TITLE
refactor: to use yarn@3 over yarn@1

### PR DESCRIPTION
**Pull request by tdeekens**

### CODE REVIEW

File: .yarnrc.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+plugins:
+  - path: .yarn/plugins/@yarnpkg/plugin-constraints.cjs
+    spec: "@yarnpkg/plugin-constraints"
+  - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
+    spec: "@yarnpkg/plugin-interactive-tools"
+  - path: .yarn/plugins/@yarnpkg/plugin-postinstall.cjs
+    spec: "https://raw.githubusercontent.com/gravitywelluk/yarn-plugin-postinstall/master/bundles/%40yarnpkg/plugin-postinstall.js"
+
+yarnPath: .yarn/releases/yarn-3.1.0.cjs
```
Comment (emmenko) [inline .yarnrc.yml:11]: Nit: 3.1.1 is latest I think

File: scripts/postinstall.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -e
+
+echo "Preparing development setup."
+yarn husky install
```
Comment (emmenko) [inline scripts/postinstall.sh:6]: Remove the `prepare` script in package.json

**Review by emmenko — COMMENTED** [review]
👌

File: .yarnrc.yml (hunk 1 above)
Comment (tdeekens) [reply .yarnrc.yml:11]: I know. I thought we'd just let bots... I'll change it.

**Review by emmenko — APPROVED** [review]

**Review by mohib0306 — APPROVED** [review]

**Pull request merged**